Zilon.Core/Zilon.BotMassLauncher/Program.cs

```diff
@@ -23,8 +23,10 @@ static void Main(string[] args)
         {
             AppDomain.CurrentDomain.ProcessExit += CurrentDomain_ProcessExit;
 
-            _pathToEnv = ConfigurationManager.AppSettings["env"];
-            _launchCount = int.Parse(ConfigurationManager.AppSettings["launchCount"]);
+            //TODO Восстановить работу с конфигами или переделать на чтение аргументов.
+            // Стало нерабочим после портирования на netcore.
+            //_pathToEnv = ConfigurationManager.AppSettings["env"];
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5028977)